helpers/aws.js

```diff
@@ -31,6 +32,10 @@ class AwsClient {
             }
         };
 
+        if (config.sessionToken) {
```

Comment:
is this added for local debugging only?

Reply:
Yes, any setup with a federated login, not sure if there is a scope to use this to move away from access key based login.

Reply:
Added this, to remove IAM user dependency  https://github.com/screwdriver-cd/screwdriver/issues/2454